src/scaffold/tests.js

```diff
@@ -0,0 +1,185 @@
+const prettier = require('prettier')
+const { strings } = require('gluegun')
+const { ascTypeForEthereum, ethereumFromAsc } = require("../codegen/types")
+
+const VARIABLES_VALUES = {
+  "i32": 123,
+  "BigInt": 234,
+  "Bytes": 1234567890,
+  "Address": "0x0000000000000000000000000000000000000001",
+  "string": "Example string value",
+  "bool": true,
+}
+
+const generateTestsFiles = (contract, events, indexEvents) => {
+  const event = events[0]
+  const eventsTypes = events.flatMap(event => event.inputs.map(input => ascTypeForEthereum(input.type))).filter(type => !type.startsWith("ethereum.") && !isNativeType(type))
```

Comment:
I appreciate that most files your text editor configuration is fixing/improving the formatting of the code 😊 
Though for this specific line it's a bit hard to read, can you split it up please? It could be something similar to this example:

```javascript
const eventsTypes = events
  .flatMap(event =>
    event
      .inputs
      .map(input => ascTypeForEthereum(input.type))
  ).filter(type => !type.startsWith("ethereum.") && !isNativeType(type))
```